doc/libpmemobj/oid_is_null.3.md

```diff
@@ -66,6 +66,8 @@ PMEMoid pmemobj_oid(const void *addr);
 uint64_t pmemobj_type_num(PMEMoid oid);
 PMEMobjpool *pmemobj_pool_by_oid(PMEMoid oid);
 PMEMobjpool *pmemobj_pool_by_ptr(const void *addr);
+void *pmemobj_direct_volatile(PMEMobjpool *pop, struct pmemvlt *vlt,
```

Comment:
This sounds too much like pmemobj_direct variant.
Maybe pmemobj_volatile?

Reply:
done